tests/python/rest_api/test_jobs.py

```diff
@@ -510,3 +515,87 @@ def test_can_export_annotations(self, admin_user: str, jobs_with_shapes: List):
         job = jobs_with_shapes[0]
         response = self._export_annotations(admin_user, job["id"], format="CVAT for images 1.1")
         assert response.data
+
+    @pytest.mark.parametrize("username, jid", [("admin1", 14)])
+    def test_check_exported_coco_annotations_structure(
+        self,
+        username,
+        jid,
+        tasks,
+        jobs,
+        annotations,
+    ):
+        job_data = jobs[jid]
+        annotations_before = annotations["job"][str(jid)]
+        task_id = job_data["task_id"]
+        task_size = tasks[task_id]["size"]
+
+        response = self._export_annotations(username, jid, format="COCO 1.0")
+        assert response.data
+        temp_anno_file_name = None
+        with NamedTemporaryFile(
+            mode="w+b", prefix="cvat", delete=False, suffix=".zip"
+        ) as temp_anno_file:
+            temp_anno_file_name = temp_anno_file.name
+            temp_anno_file.write(response.data)
+
+        with zipfile.ZipFile(temp_anno_file_name, mode="r") as zip_file:
+            exported_annotations = json.loads(zip_file.read(zip_file.namelist()[0]))
+
+        assert len(annotations_before["shapes"]) == len(exported_annotations["annotations"])
+        # NOTE: data step is not stored in assets, default = 1
+        assert job_data["stop_frame"] - job_data["start_frame"] + 1 == len(
+            exported_annotations["images"]
+        )
+
+        assert task_size > len(exported_annotations["images"])
+        os.remove(temp_anno_file_name)
+
+    @pytest.mark.parametrize("username, jid", [("admin1", 14)])
+    def test_check_exported_cvat_dataset_structure(
```

Comment:
Why only these two formats were tested?

Reply:
> Why only these two formats were tested?

I believe that in these tests 2 formats will be enough because tests should check that the annotation file contains only job annotations (the dataset contains only job data) and not the entire task. If this is true for our format, then the same for the others.